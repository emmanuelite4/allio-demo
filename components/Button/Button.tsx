import styled from "@emotion/styled";
import { CSS_BUTTON } from "../../styles/shared";

const Button = styled("button")`
  ${CSS_BUTTON};
  font-size: 18px;
  height: 44px;
  width: 44px;
`;

export default Button;
