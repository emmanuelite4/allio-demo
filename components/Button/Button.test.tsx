import Button from "./Button";
import renderer from "react-test-renderer";

describe("Button", () => {
  it("matches snapshot", () => {
    const buttonEl = renderer.create(<Button>hi</Button>).toJSON();
    expect(buttonEl).toMatchSnapshot();
  });
});
