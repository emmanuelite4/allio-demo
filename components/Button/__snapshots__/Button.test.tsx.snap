// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button matches snapshot 1`] = `
.emotion-0 {
  background-color: var(--color-primary);
  border-radius: 10px;
  outline: none;
  border: none;
  color: #ffffff;
  font-weight: bold;
  cursor: pointer;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  font-size: 18px;
  height: 44px;
  width: 44px;
}

<button
  className="emotion-0"
>
  hi
</button>
`;
